src/api/api.go

```diff
@@ -146,6 +146,7 @@ type CreateGroupResponse struct {
 type UpdateProfileRequest struct {
 	Name         string `json:"name"`
 	Base64Avatar string `json:"base64_avatar"`
+	About 			 string `json:"about"`
```

Comment:
I think I would prefer a pointer here. i.e: `About 			 *string `json:"about"`
(Here's an example of what I meant: https://github.com/bbernhard/signal-cli-rest-api/blob/master/src/api/api.go#L57 and https://github.com/bbernhard/signal-cli-rest-api/blob/master/src/client/client.go#L1639-L1641)

The reason for that is, that I am not sure if there is a difference in the Signal UI between not setting `about` and setting it to an empty string. By using a pointer we could check if the value is actually set, before passing the `about` string to the underlying `signal-cli` command.

Reply:
Makes sense. I did test it with adding the about and I could not see any difference in the UI, but I think it still makes sense to conditionally send it to signal-cli.

I've tested it with the new changes, with and without the about text in native mode, and it works as expected.